.idea/encodings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can this (`.idea`) be removed?  I think you removed it at some point and the re-added it